packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -181,8 +182,45 @@ export interface VideoGalleryProps {
    * This callback will be called when a participant video tile is un-pinned
    */
   onUnpinParticipant?: (userId: string) => void;
+  /* @conditional-compile-remove(pinned-participants) */
+  /**
+   * Options for showing the remote video tile menu.
+   * default: \{ type: 'contextual' \}
+   */
+  remoteVideoTileMenuOptions?: false | VideoTileContextualMenuProps | VideoTileDrawerMenuProps;
 }
 
+/* @conditional-compile-remove(pinned-participants) */
+/**
+ * Properties for showing contextual menu for remote {@link VideoTile} components in {@link VideoGallery}.
+ *
+ * @beta
+ */
+export interface VideoTileContextualMenuProps {
+  /**
+   * The menu property type
+   */
+  type: 'contextual';
+}
+
+/* @conditional-compile-remove(pinned-participants) */
+/**
+ * Properties for showing drawer menu on remote {@link VideoTile} long touch in {@link VideoGallery}.
+ *
+ * @beta
+ */
+export interface VideoTileDrawerMenuProps {
+  /**
+   * The menu property type
+   */
+  type: 'drawer';
+  /**
+   * The optional id property provided on an element that the drawer menu should render within when a
+   * remote participant video tile is long touched. If an id is not provided, then a drawer menu will
```

Comment:
remove `is long touched` here in this comment, that behavior isn't specific to this property and could change
```suggestion
  * remote participant video tile drawer menu is shown. If an id is not provided, then a drawer menu will
```